### ISSUE TITLE
Can we show the calculation/reward scheme for each token?

### ISSUE THREAD

**Issue by envystakepool** [FreeLoaderz/SmartClaimzBugs#6]
Can we show the calculation/reward scheme for each token? So if PSPO equation is R = (S - 150)^0.6 - 50, we could display that for people?

**Comment by santonode** [comment]
We did put this in the FAQ

**Comment by EtherealStakePool** [comment]
Yes, but I'm wondering about having it visible as part of the Claimz? I don't think it's the most intuitive to have it on the FAQ page? It would be nice to see it visible with the token you are claiming if possible, but just a thought.

**Comment by EtherealStakePool** [comment]
Or have a separate page/tab that says 'Token Distribution Method' or something like that? That just displays a table with each token?

**Comment by santonode** [comment]
Sounds good to me. Steve would probably need a bit more details on what we want.

**Comment by fisherstevenk** [comment]
It could be added to a token "metadata" page. 

Basically, first view is like the pool listing.  Then you click a token and get the single view.  The single view could have the expanded data.

**Comment by santonode** [comment]
Works for me.
